dotnet-interactive/resources/code-mirror-lsp.js

```diff
@@ -0,0 +1,131 @@
+﻿define({
+    init: function (global) {
+        // used to display textDocument/hover
+        var hoverDiv = document.createElement('div');
```

Comment:
I think document should be passed in or looked up onto global parameter, this code could be hard to test without always running inside a browser.

Reply:
Good call, will do.